documentation/src/main/asciidoc/public/reference/_search-dsl-predicate.adoc

```diff
@@ -1865,6 +1865,47 @@ or for the whole predicate with a call to `.boost(...)` after `.matching(...)`.
 of targeted fields to analyze searched text by default,
 but this can be <<search-dsl-predicate-common-overriding-analysis,overridden>>.
 
+[[search-dsl-predicate-prefix]]
+== `prefix`: match documents based on what a field starts with
+
+The `preifx` predicate matches documents for which a given field starts with a prefix matching the given string.
```

Comment:
Typo.

Also "starts with a prefix matching the given string" just means "starts with a given string", no?

```suggestion
The `prefix` predicate matches documents for which a given field has a value starting with a given string.
```